client_example/20_splitk_gemm/splitK_gemm_bf16_i8.cpp

```diff
@@ -0,0 +1,238 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Hi, Jing, do you have a plan to add some tuning instances and a client example for bf16_bf16 split-k gemm algo?